docs/tutorials/depth-reduction-with-circuit-cutting.ipynb

```diff
@@ -136,7 +136,7 @@
    "id": "68c00476",
    "metadata": {},
    "source": [
-    "![swaps-2.png](https://learning-api.quantum.ibm.com/assets/ce74853a-8e5b-460b-9dc6-69eeadca2d05)"
+    "![swaps.png](/docs/images/tutorials/depth-reduction-with-circuit-cutting/swaps.avif)"
```

Comment:
I opened https://github.com/Qiskit/documentation/issues/3263 to fix this alt text